addons/fleet/migrations/12.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,161 @@
+DEL fleet.service.type: fleet.type_service_service_2
+DEL fleet.service.type: fleet.type_service_service_3
+DEL fleet.service.type: fleet.type_service_service_5
+DEL fleet.service.type: fleet.type_service_service_6
+DEL fleet.service.type: fleet.type_service_service_7
+DEL fleet.service.type: fleet.type_service_service_8
+DEL fleet.service.type: fleet.type_service_service_9
+DEL fleet.vehicle.model: fleet.model_a1
+DEL fleet.vehicle.model: fleet.model_a3
+DEL fleet.vehicle.model: fleet.model_a4
+DEL fleet.vehicle.model: fleet.model_a5
+DEL fleet.vehicle.model: fleet.model_a6
+DEL fleet.vehicle.model: fleet.model_a7
+DEL fleet.vehicle.model: fleet.model_a8
+DEL fleet.vehicle.model: fleet.model_agila
+DEL fleet.vehicle.model: fleet.model_ampera
+DEL fleet.vehicle.model: fleet.model_antara
+DEL fleet.vehicle.model: fleet.model_astra
+DEL fleet.vehicle.model: fleet.model_astragtc
+DEL fleet.vehicle.model: fleet.model_classa
+DEL fleet.vehicle.model: fleet.model_classb
+DEL fleet.vehicle.model: fleet.model_classc
+DEL fleet.vehicle.model: fleet.model_classcl
+DEL fleet.vehicle.model: fleet.model_classcls
+DEL fleet.vehicle.model: fleet.model_classe
+DEL fleet.vehicle.model: fleet.model_classgl
+DEL fleet.vehicle.model: fleet.model_classglk
+DEL fleet.vehicle.model: fleet.model_classm
+DEL fleet.vehicle.model: fleet.model_classr
+DEL fleet.vehicle.model: fleet.model_classs
+DEL fleet.vehicle.model: fleet.model_classslk
+DEL fleet.vehicle.model: fleet.model_classsls
+DEL fleet.vehicle.model: fleet.model_combotour
+DEL fleet.vehicle.model: fleet.model_corsa
+DEL fleet.vehicle.model: fleet.model_insignia
+DEL fleet.vehicle.model: fleet.model_meriva
+DEL fleet.vehicle.model: fleet.model_mokka
+DEL fleet.vehicle.model: fleet.model_q3
+DEL fleet.vehicle.model: fleet.model_q5
+DEL fleet.vehicle.model: fleet.model_q7
+DEL fleet.vehicle.model: fleet.model_serie1
+DEL fleet.vehicle.model: fleet.model_serie3
+DEL fleet.vehicle.model: fleet.model_serie5
+DEL fleet.vehicle.model: fleet.model_serie6
+DEL fleet.vehicle.model: fleet.model_serie7
+DEL fleet.vehicle.model: fleet.model_seriehybrid
+DEL fleet.vehicle.model: fleet.model_seriem
+DEL fleet.vehicle.model: fleet.model_seriex
+DEL fleet.vehicle.model: fleet.model_seriez4
+DEL fleet.vehicle.model: fleet.model_tt
+DEL fleet.vehicle.model: fleet.model_zafira
+DEL fleet.vehicle.model: fleet.model_zafiratourer
+# Keep those data, do not remove it
+
+NEW fleet.vehicle.state: fleet.fleet_vehicle_state_new_request (noupdate)
+# NOTHING TO DO
+
+DEL fleet.vehicle.tag: fleet.vehicle_tag_break
+DEL fleet.vehicle.tag: fleet.vehicle_tag_compact
+DEL fleet.vehicle.tag: fleet.vehicle_tag_convertible
+DEL fleet.vehicle.tag: fleet.vehicle_tag_junior
+DEL fleet.vehicle.tag: fleet.vehicle_tag_leasing
+DEL fleet.vehicle.tag: fleet.vehicle_tag_purchased
+DEL fleet.vehicle.tag: fleet.vehicle_tag_sedan
+DEL fleet.vehicle.tag: fleet.vehicle_tag_senior
+# Keep those data, do not remove it
```

Comment:
Why don't you group all the entries by logic operation?

Reply:
What do you mean?

Reply:
In the work file, cut these lines and paste them in the same block than the rest with the same comment (`# Keep those data, do not remove it`)